src/main/resources/hudson/plugins/ws_cleanup/WsCleanup/config.jelly

```diff
@@ -45,5 +45,8 @@ DEALINGS IN THE SOFTWARE. -->
         <f:entry title="${%External Deletion Command}" help="/plugin/ws-cleanup/help/externalDelete.html">
             <f:textbox field="externalDelete" />
         </f:entry>
+        <f:entry title="${%Clean asynchronously}" help="/plugin/ws-cleanup/help/cleanAsynchronously.html">
+            <f:checkbox field="cleanAsynchronously" checked="${it.cleanAsynchronously}" />
```

Comment:
We always reboot our slaves after a build and the async delete makes it hard to determine when the clean has finished, would be in favor of providing an option to enable a sync clean, that way we can reboot after the wscleanup step safely